### PR TITLE
Improved Profile Popups to Material UI

### PR DESCRIPTION
**Pull request by theprover97**
The popups associated with Profile page have been enhanced using Material UI.

### CODE REVIEW

**Comment by theprover97** [comment]
Edit-Profile

**Comment by theprover97** [comment]
Followers

**Comment by theprover97** [comment]
Logout

**Comment by theprover97** [comment]
Actually this code is from a separate branch. Hence it does not have the logout part.

**Comment by theprover97** [comment]
Yeah, I have added InputLabel tag rather than InputProps and on focus all input fields have their labels shrunk to top.

**Comment by Rupeshiya** [comment]
@theprover97  Please resolve the conflicts.
Then you are good to go.

**Comment by theprover97** [comment]
@Rupeshiya I have resolved the merge conflicts and squashed the commits.

File: src/user/profile/popups/followers.js [hunk 1]
```diff
@@ -8,74 +12,168 @@ export class Followers extends Component{
...
+        <Dialog
+          {...this.props}
+          aria-labelledby="customized-dialog-title">
+          <DialogTitle> 
+            <span className="title">Followers</span>
+            {this.props.onClose ? <IconClose/> : null}
+            <div className="about">PEOPLE WHO FOLLOW YOU</div>
+          </DialogTitle> 
+          <DialogContent>
+            <Grid container spacing={4}>
```
Comment (Rupeshiya) [inline src/user/profile/popups/followers.js]: @theprover97 Just use a for loop, store name and designation object in an array.
We can reduce the repetition of the code.
Hope you understand what I am trying to say.

**Comment by theprover97** [comment]
Sure @Rupeshiya . Will do that. Thanks!

**Comment by theprover97** [comment]
@Rupeshiya currently I am only displaying name and designation of single person. So, will it be acceptable that I write them directly in the code, rather than forming arrays for them. I can add it later when we write for actual data.

**Comment by Rupeshiya** [comment]
@theprover97 According to me, It would be great if we do these things from now itself.
What say @vaibhavdaren ?

**Comment by theprover97** [comment]
@Rupeshiya @vaibhavdaren I have made the changes. Please give your review.

File: src/user/profile/popups/followers.js [hunk 2]
```diff
@@ -1,81 +1,83 @@
...
+      
+      let followers = customNotifications.map((a) => {
+        return(
+          <Grid container spacing={4}>        
+          <Grid item sm={2} xs={3 | 12}>
+            <img class="photo" src={logo} alt="I"></img>
+          </Grid>
+          <Grid item sm={7} xs={9 | 12}>
+            <h5><b>{a.heading}</b></h5>
+            <p className="descr">{a.content}</p>
```
Comment (Rupeshiya) [inline src/user/profile/popups/followers.js]: @theprover97 Make it ellipsis when the content is of large size, as you have used the notification data's content.
Else everything is fine 
Keep up the good work!

**Comment by theprover97** [comment]
@Rupeshiya I have shortened the user description using substring. We can use ellipsis later when adding user description to DB.

**Pull request merged**